src/test/scala/it/unibo/authsim/library/dsl/policy/builders/PolicyOptionalBuilderTests.scala

```diff
@@ -12,7 +12,7 @@ import org.scalatest.matchers.*
 
 import scala.language.postfixOps
 
-class PolicyBuilderTests extends AnyFlatSpec with should.Matchers:
+class PolicyOptionalBuilderTests extends AnyFlatSpec with should.Matchers:

```

Comment:
Also this class has been renamed, why?

Reply:
Ok, I definitly have launch a rename and I accidentally to rename all the elements that contain that string. I am sorry.